src/index.ts

```diff
@@ -1,10 +1,12 @@
 /* eslint-disable no-console */
 import pnldDownload from "./pnld-download"
+import requestedChangesDownload from "./requested-changes-download"
 
 const main = async () => {
   await pnldDownload()
+  await requestedChangesDownload()
```

Comment:
what does this actually do?

Reply:
no, its fine. I was just confused as I could not find the definition for the function in this PR. That is why I was confused

Reply:
oh, I see now! it's https://github.com/ministryofjustice/bichard7-next-data/blob/alice/BICAWS-1831-requested-changes/src/requested-changes-download/index.ts